paper/paper.tex

```diff
@@ -753,8 +761,8 @@ \subsubsection{Julia is fast}
 Julia is not generically slower than traditional high-performance programming
 languages like C, C++, and Fortran. A minor exception is due to
 Julia being built on top of LLVM, which does not perform the same optimizations as other
-C/C++/Fortran compilers. In particular, LLVM does
-not use fused multiply-add (FMA) instructions by default, while GCC and Intel
+C/C++/Fortran compilers. In particular, most LLVM frontends (including Julia) do
+not enable fused multiply-add (FMA) contraction by default, while GCC and Intel
```

Comment:
I still think this is misleading: it has nothing to do with being built on top of LLVM, simply that it is an optimization that isn't enabled by default (the fact that LLVM-based frontends tend not to enable it by default is perhaps interesting, but coincidental).

Reply:
Do you have a suggestion how to rewrite this part accurately?

Reply:
I wrote another version in 4138f40, hoping to clarify this aspect.

Reply:
I would suggest something along the lines of the following: 

> Julia doesn't perform automatic fused multiply-add (FMA) contraction, that is replacing a multiplication followed by an addition with a single FMA instruction, as it is inconsistent with its strict floating point semantics. In other compilers, this optimization is controlled by a global compiler option (`-ffp-contract` in GCC and Clang). In Julia, it is controlled more locally: the `muladd` intrinsic function allows the compiler to evaluate a multiply-add using the most efficient method, and the `@muladd` macro will syntactically rewrite expressions to insert the `muladd` function in appropriate locations.

Reply:
> I wrote another version in 4138f40, hoping to clarify this aspect. Does this look better to you, @simonbyrne?

That also looks good. Thanks, I had started my reply before you posted yours, but yours addresses my main concern.

Reply:
Your suggestion gives additional information and is really helpful :+1: Thus, i rewrote this paragraph again in 7cbbf6c, merging your suggestion and my previous version.